charts/function-mesh-operator/Chart.yaml

```diff
@@ -34,3 +34,6 @@ dependencies:
     condition: admissionWebhook.enabled
     version: 0.2.6
 #    repository: https://charts.functionmesh.io/
+  - name: vertical-pod-autoscaler
+    repository: https://cowboysysop.github.io/charts/
```

Comment:
Is there any official charts for VPA? I noticed that the one this PR uses is not an official one, see https://github.com/cowboysysop/charts/tree/master/charts/vertical-pod-autoscaler

Reply:
there is no official charts, the official are using shell scripts to install vpa